### ISSUE TITLE
How to use behind Corporate Proxy

### ISSUE THREAD

**Issue by wabmca** [nodejitsu/node-http-proxy#541]
I am using express behind enterprise corporate proxy/firewall. 
I am struggling because because I just don't know where to set proxy for node.js

I usually export http_proxy="http://<username>:<password>@myproxy.com:8080 at prompt. This is just sufficient. 

But this setting is not working for node.js. 
Please help.
Should I use node-http-proxy if yes then please give a simple code sample for my code request. 
Else just give a sample node.js to do this. 

Thanks in advance.

**Comment by ultimate-tester** [comment]
No, this is not the right thing for you. What are you trying to do with Node that requires a proxy? What module are you using that is creating those http requests?
